network.go

```diff
@@ -574,9 +580,91 @@ func (s *sandbox) updateRoute(netHandle *netlink.Handle, route *types.Route, add
 /////////
 // DNS //
 /////////
+func mountFile(source, destination, fsType string, flags int, options string) error {
```

Comment:
I believe there exists a function to perform bindmount already in the agent repo. Can you please reuse that?